plugins/text-freetype2/CMakeLists.txt

```diff
@@ -11,27 +11,49 @@ endif()
 
 find_package(Freetype REQUIRED)
 
+if(OS_MACOS)
+  find_package(Iconv REQUIRED)
+elseif(NOT OS_WINDOWS)
+  find_package(Fontconfig REQUIRED)
+endif()
+
 add_library(text-freetype2 MODULE)
 add_library(OBS::text-freetype2 ALIAS text-freetype2)
 
-target_sources(text-freetype2 PRIVATE find-font.h obs-convenience.c text-functionality.c text-freetype2.c
-                                      obs-convenience.h text-freetype2.h)
-target_link_libraries(text-freetype2 PRIVATE OBS::libobs Freetype::Freetype)
+target_sources(
+  text-freetype2
+  PRIVATE # cmake-format: sortable
+          $<$<PLATFORM_ID:Darwin>:find-font-cocoa.m>
+          $<$<PLATFORM_ID:Darwin>:find-font-iconv.c>
+          $<$<PLATFORM_ID:Darwin>:find-font.c>
+          $<$<PLATFORM_ID:Linux,FreeBSD>:find-font-unix.c>
+          $<$<PLATFORM_ID:Windows>:find-font-windows.c>
+          $<$<PLATFORM_ID:Windows>:find-font.c>
+          find-font.h
+          obs-convenience.c
+          obs-convenience.h
+          text-freetype2.c
+          text-freetype2.h
+          text-functionality.c)
+
+# cmake-format: off
+target_link_libraries(
+  text-freetype2
+  PRIVATE OBS::libobs
+          Freetype::Freetype
+          $<$<PLATFORM_ID:Darwin>:Iconv::Iconv>
+          "$<$<PLATFORM_ID:Darwin>:$<LINK_LIBRARY:FRAMEWORK,Cocoa.framework>>"
+          $<$<PLATFORM_ID:Linux,FreeBSD>:Fontconfig::Fontconfig>)
+# cmake-format: on
+
+# freetype is hardcoded with /DEFAULTLIB:MSVCRT
```

Comment:
Is this our fault when building? Or is this in the dependency itself?

Reply:
I think I haven't found an official configuration option to change/adjust this in `freetype`.

Reply:
Well, the comment here will have to suffice as a sort of TODO for us for later.